packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,86 @@
+.spinner {
+  position: relative;
+  pointer-events: none;
+  /* 1 */
+  opacity: 0.6;
+}
+
+.spinner:after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  z-index: 3;
+  width: 2.25rem;
+  height: 2.25rem;
```

Comment:
We should convert the sizes in here to our space units.

https://atlantis.getjobber.com/space

Some examples of this here:
https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Switch/Switch.css

Reply:
Excellent, using space units now!